### PR TITLE
Fixed plugin URL should be translatable

### LINKED ISSUE #11983: URL to Classic Editor plugin should be translatable
**Describe the bug**
In /gutenberg/tags/4.4.0/gutenberg.php there's a link to the Classic Editor plugin. It should be translatable, so that I can point to the Swedish version in the Swedish translation.

```
// Translators: link is for Classic Editor plugin.
--
__( 'The Block Editor requires JavaScript. Please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
'https://wordpress.org/plugins/classic-editor/'
);

```

### PR DESCRIPTION
**Pull request by dixitadusara**
## Description
I have submitted PR as per your suggestions. Let me know If any changes require.

Previously: #11992

Fixes #11983.

### CODE REVIEW

**Comment by swissspidy** [comment]
There was no need to create a new PR just to address the feedback, but thanks for incorporating it.

**Review by swissspidy — APPROVED** [review]

**Review by swissspidy — CHANGES_REQUESTED** [review]
There's currently PHPCS error because of the translator comment.

File: gutenberg.php [hunk 1]
```diff
@@ -41,9 +41,9 @@ function the_gutenberg_project() {
 			);
 		} else { // Using Gutenberg in Core.
 			printf(
-				// Translators: link is for Classic Editor plugin.
+				// Translators: %s: https://wordpress.org/plugins/classic-editor/
```
[the remark this answers is not in the document]
Comment (dixitadusara) [reply gutenberg.php]: May I create a new PR for this change? Please guide me.

[the remark this answers is not in the document]
Comment (swissspidy) [reply gutenberg.php]: @dixitadusara No, you don't need to create a new PR. You can just add this as a new commit to this existing PR by clicking on the "Commit suggestion" button.

See https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change for how this suggestion feature works.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**